### PR TITLE
Misc styling

### PR DESCRIPTION
**Pull request by martipal**
- comment count has fixed width
- changed userfeed refresh button
- added title to username on hover
- username displayed shortened if display name > 14 characters
- remove pop-up when user image clicked 
- when creating a new group, new group appears on the top of groups list
- removed left margin from song logs

### CODE REVIEW

File: sound-bubble/imports/ui/stylesheets/main.css [hunk 1]
```diff
@@ -519,8 +506,10 @@ padding:0;
   border:1px solid #34363b;
   float:right;
   background:url(https://media1.giphy.com/media/l3q2Xl2Rd8wZfMCaI/200w.webp?cid=790b76115d3e6c3732544e6577a43431&rid=200w.webp) repeat;
+  overflow-wrap:break-word;
 }
```
Comment (Talos6) [inline sound-bubble/imports/ui/stylesheets/main.css]: Comments shows when hover the user icon components, but user icon only gets bigger when hover inside the padding. There should be a 0 padding or 0 margin to keep the icon enlarge when hover and show the comments

**Comment by veronicz** [comment]
In my browser, username on hover is cropped to the width of photo_username so it doesn't always display the full username (I tried with soundbubbletesting). Is there a way to make it float on top of everything else?

File: sound-bubble/imports/ui/stylesheets/main.css (hunk 1 above)
Comment (veronicz) [reply sound-bubble/imports/ui/stylesheets/main.css]: This seems to be caused by "margin-right: 1em;" on user-comment-image, but removing it changes the comments alignment a bit.

Comment (martipal) [reply sound-bubble/imports/ui/stylesheets/main.css]: fixed

**Pull request merged**